docs/CNS.md

```diff
@@ -3,100 +3,123 @@
 
-## 1 Download CNS
+Before starting, make sure you download the system requirements, they are `gcc`, `gfortran` and `csh`. Luckily they are easily available from the package manager, so if you are using Ubuntu (for example):
 
-Downloading CNS requires a license, which academic users can [request for free](http://cns-online.org) (click on the `Download` menu and fill the form). You will be sent a password for the download via e-mail.
-Using the link provided in the same e-mail, download CNS ("Source installer (compilation required on your system)") to the directory in which you want to install it (for example `software`). Navigate to this directory, then uncompress and extract the archive:
+```bash
+sudo apt-get install gcc gfortran csh
+```
+
+## 1. Downloading CNS
+
+Downloading CNS requires a license, which academic users can [request for free](http://cns-online.org).
+
+To Download CNS, go to their [website](http://cns-online.org) and  click on the `Download` menu and fill the form. You will be sent a password for the download via e-mail - keep in mind only education e-mails are allowed for this.
+
+You will receive a login/password in the e-mail, once you have those go to the [download page](http://cns-online.org/download/v1.3/cns_solve_1.3_all.tar.gz) and use the provided login/password to download the CNS package.
+
+> The CNS website is not currently using SSL certificates, this mean your download might be blocked or it might give you a warning, double check in your browser.
+
+Once the archive `cns_solve_1.3_all.tar.gz` has been downloaded, move it to a `$HOME/software` directory (or any other directory of your choice) and decompress it:
 
 ```bash
-cd ~/software/
-gunzip cns_solve_1.3_all.tar.gz
-tar -xvf cns_solve_1.3_all.tar
+mkdir ~/software
+mv ~/Downloads/cns_solve_1.3_all.tar.gz ~/software/
+tar -zxvf cns_solve_1.3_all.tar.gz
 ```
 
+The command above will have create a `cns_solve_1.3` directory in the `~/software` directory, containing the CNS source code.
+
+## 2. Configuring CNS
+
 Go to the CNS directory:
 
 ```bash
 cd cns_solve_1.3
 ```
 
-If you are using csh/tcsh, edit `cns_solve_env` to point to the correct location (the current directory):
+Inside this directory there will be a `cns_solve_env` file.
 
-```
-# CHANGE THE NEXT LINE TO POINT TO THE LOCATION OF THE CNSsolve DIRECTORY
+You need to edit this file to point to the current directory.
 
-            setenv CNS_SOLVE /PATH/TO/cns_solve_1.3
-```
+You will need to change line 18 of this file to point to the current directory.
 
-If you are using sh/bash, edit `.cns_solve_env_sh`:
+You can do that either by opening this file in a code editor such as VScode or vim, or by running the command below:
 
-```
-# CHANGE THE NEXT LINE TO POINT TO THE LOCATION OF THE CNSsolve DIRECTORY
-
-        CNS_SOLVE=/PATH/TO/cns_solve_1.3
+```bash
+sed -i 's@_CNSsolve_location_@'"$PWD"'@' cns_solve_env
```

Comment:
Should we add ` sed -i '' 's@_CNSsolve_location_@'"$PWD"'@‘ cns_solve_env ` as an alternative to installing `gnu-sed` on Mac? 
But also idk if it works in general or only ‘on my machine’